### PR TITLE
Added smooth scroll when clicking on navtabs for screen-body.

### PR DESCRIPTION
**Pull request by jalopez13**
Issue :https://github.com/wb-apps/tmz-cms/issues/611

Added smooth scroll to top when navigating via tab links. Ready for code review.

### CODE REVIEW

**Comment by efjacobson** [comment]
@jalopez13 next time Greg is in the office please demo this for us and Mariam

**Comment by jalopez13** [comment]
@efjacobson - Updated code to use react ref instead of screen-body ID. Ready for CR.

File: src/components/screen/index.jsx [hunk 1]
```diff
@@ -22,9 +22,12 @@ class Screen extends React.Component {
       isSidebarOpen: true,
       isSidenavOpen: true,
     };
+    
+    this.screenBodyRef = React.createRef();
 
     this.toggleSidebar = this.toggleSidebar.bind(this);
     this.toggleSidenav = this.toggleSidenav.bind(this);
+    this.smoothScroll = this.smoothScroll.bind(this);
```
Comment (efjacobson) [inline src/components/screen/index.jsx]: lets do `scrollToTop`

File: src/components/screen/index.jsx [hunk 2]
```diff
@@ -41,12 +44,18 @@ class Screen extends React.Component {
...
+    this.smoothScroll();
   }
 
   componentWillUpdate(nextProps) {
     document.title = nextProps.title || 'Triniti';
   }
 
+  smoothScroll() {
+    const screenBody = this.screenBodyRef.current;
+    screenBody.scrollTo({ top: 0, behavior: 'smooth' });
```
Comment (efjacobson) [inline src/components/screen/index.jsx]: lets do `behavior: auto` here

**Comment by efjacobson** [comment]
@gdbrown this is ready for release, want me to prep?

**Pull request merged**